systemtests/utils/utils.go

```diff
@@ -72,55 +72,60 @@ func startNetPlugin(t *testing.T, nodes []VagrantNode, nativeInteg bool) {
 
 	for i, node := range nodes {
 		//start the netplugin
-		cmdStr := fmt.Sprintf("sudo PATH=$PATH nohup netplugin %s -host-label host%d 0<&- &>/tmp/netplugin.log &", nativeIntegStr,
-			i+1)
-		output, err := node.RunCommandWithOutput(cmdStr)
```

Comment:
For running node commands for dind Output() always seems to block forever. I spent a lot of time on trying different things but could not make it work other than converting that to Run(). We can try together tomorrow to see if you have any ideas.